web/src/main/java/se/inera/intyg/webcert/web/csintegration/certificate/IntegratedUnitHelper.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) 2024 Inera AB (http://www.inera.se)
+ *
+ * This file is part of sklintyg (https://github.com/sklintyg).
+ *
+ * sklintyg is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * sklintyg is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package se.inera.intyg.webcert.web.csintegration.certificate;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+import se.inera.intyg.infra.security.common.model.UserOriginType;
+import se.inera.intyg.webcert.web.integration.registry.IntegreradeEnheterRegistry;
+import se.inera.intyg.webcert.web.integration.registry.dto.IntegreradEnhetEntry;
+import se.inera.intyg.webcert.web.service.user.WebCertUserService;
+
+@Component
+@RequiredArgsConstructor
+public class IntegratedUnitHelper {
+
+    private final WebCertUserService webCertUserService;
+    private final IntegreradeEnheterRegistry integreradeEnheterRegistry;
+
+    public void registerUnit() {
```

Comment:
Nice helper method to handle storage of integrated units. Two questions

1. If we change the name of the class to IntegratedUnitRegistryHelper, does it make the purpose of the class clearer? If you agree, maybe update the method-name to register.
2. Would be nice if we could use this helper in ```CreateDraftCertificateFromCS``` aswell (on line 78), do you think its worth refactoring so that we can handle ```IntygUser``` aswell?